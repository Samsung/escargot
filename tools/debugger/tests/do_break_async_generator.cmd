s
s
s
bt
s
s
s
s
b do_break_async_generator.js:21
c
bt
c
bt
c
