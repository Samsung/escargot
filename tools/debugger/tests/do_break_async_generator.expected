Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_break_async_generator.js:38
(escargot-debugger) s
Stopped at tools/debugger/tests/do_break_async_generator.js:39
(escargot-debugger) s
Stopped at tools/debugger/tests/do_break_async_generator.js:33 (in g() at line:32, col:1)
(escargot-debugger) s
Stopped at tools/debugger/tests/do_break_async_generator.js:21 (in f1() at line:20, col:1)
(escargot-debugger) bt
tools/debugger/tests/do_break_async_generator.js:21:5 [depth:0] (in f1)
tools/debugger/tests/do_break_async_generator.js:38:10
(escargot-debugger) s
Stopped at tools/debugger/tests/do_break_async_generator.js:44
(escargot-debugger) s
Stopped at tools/debugger/tests/do_break_async_generator.js:42 (in h() at line:41, col:1)
(escargot-debugger) s
Stopped at tools/debugger/tests/do_break_async_generator.js:43 (in h() at line:41, col:1)
(escargot-debugger) s
Stopped at tools/debugger/tests/do_break_async_generator.js:46
(escargot-debugger) b do_break_async_generator.js:21
Breakpoint 1 at tools/debugger/tests/do_break_async_generator.js:21 (in f1() at line:20, col:1)
(escargot-debugger) c
Stopped at breakpoint:1 tools/debugger/tests/do_break_async_generator.js:21 (in f1() at line:20, col:1)
(escargot-debugger) bt
tools/debugger/tests/do_break_async_generator.js:21:5 [depth:0] (in f1)
tools/debugger/tests/do_break_async_generator.js:25:12 [depth:1] (in f2)
tools/debugger/tests/do_break_async_generator.js:38:10
(escargot-debugger) c
Stopped at breakpoint:1 tools/debugger/tests/do_break_async_generator.js:21 (in f1() at line:20, col:1)
(escargot-debugger) bt
tools/debugger/tests/do_break_async_generator.js:21:5 [depth:0] (in f1)
tools/debugger/tests/do_break_async_generator.js:25:12 [depth:1] (in f2)
tools/debugger/tests/do_break_async_generator.js:29:12 [depth:2] (in f3)
tools/debugger/tests/do_break_async_generator.js:38:10
(escargot-debugger) c
Connection closed.
